src/Sticky.js

```diff
@@ -1,39 +1,39 @@
-import React, { Component } from "react";
-import ReactDOM from "react-dom";
-import PropTypes from "prop-types";
+import React, { Component } from 'react';
```

Comment:
I think your formatter has different Prettier settings. I used defaults for the previous pass.

Reply:
Ah, was wondering about this.